src/doc/template/tmpl/layout.tmpl

```diff
@@ -74,7 +74,7 @@
         <link type="text/css" rel="stylesheet" href="styles/demos.css">
         <link type="text/css" rel="stylesheet" href="styles/prism.css"  />
         <link type="text/css" rel="stylesheet" href="styles/britecharts.css" />
-        <script src="/britecharts/scripts/common.js"></script>
```

Comment:
AFAIK this is necessary for the docs to work when uploaded to the documentation page. We are using a proxy in dev: https://github.com/eventbrite/britecharts/blob/master/webpack.config.js#L20

Reply:
Ah, alright. I missed the local server task and was having issues loading the static files directly. Other assets (i.e. CSS) are referenced relatively, but I can revert this change if it will cause issues.

Reply:
Did you try to run it locally and it works?

Reply:
Yeah, I think it will be better to revert this changes.

Thanks a lot!